src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Batch.cs

```diff
@@ -175,17 +184,25 @@ public BatchSummary Summary
         {
             get
             {
-                return new BatchSummary
+                // Basic summary with basic information
```

Comment:
Minor: that's a lot of "basics" in 1 comment. Just "basic summary with key information"?